scripts/release/steps/clean-changelog-unreleased.js

```diff
@@ -0,0 +1,75 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+const execa = require("execa");
+const { outdent } = require("outdent");
+const { logPromise } = require("../utils");
+
+async function clean() {
+  const changelogUnreleasedDir = path.join(
+    __dirname,
+    "../../../changelog_unreleased"
+  );
+
+  // Revert blog-post-intro.md
+  const introFile = path.join(changelogUnreleasedDir, "blog-post-intro.md");
+  fs.writeFileSync(
+    introFile,
+    outdent`
+  ---
+  author: "🚧"
+  authorURL: "https://github.com/🚧"
+  title: "Prettier 🚧"
+  ---
+
+  🚧 Write an introduction here.
+
+`
+  );
```

Comment:
On draft-blog-post step, copy `blog-post-intro.md` to `blog-post-intro-2.2.0.md`, use if, and in this step remove `blog-post-intro-2.2.0.md`

Reply:
Oh, in that step, there is no version info, so `blog-post-intro-releasing.md` or something.

Reply:
Do you mean to create a draft blog post in the release script?

Reply:
Update "scripts/draft-blog-post.js"

```js
const introFile = path.join(changelogUnreleasedDir, "blog-post-intro-releasing.md");
if (!fs.exitsSync(introFile)) {
  // Copy "blog-post-intro.md" to "blog-post-intro-releasing.md"
}
```

Here, just delete `changelog_unreleased/blog-post-intro-releasing.md` file.

Meaning we don't use "blog-post-intro.md" file directly, it's used as a template

Reply:
I've added two changes:
1. Add `BLOG_POST_INTRO_TEMPLATE.md` and make` blog-post-intro.md` a file that can be edited directly.
2. Remove `clean-changelog-unreleased` from the release script to make it a standalone script.